docs/buildah-config.md

```diff
@@ -41,33 +41,33 @@ edits") in any images which will be created using the specified container.
 
 **--entrypoint** *entry*
 
-Sets the *entry point* for containers based on any images which will be built
+Set the *entry point* for containers based on any images which will be built
 using the specified container.
 
 **--env** *var=value*
 
-Adds a value (e.g. name=*value*) to the environment for containers based on any
-images which will be built using the specified container.
+Add a value (e.g. name=*value*) to the environment for containers based on any
+images which will be built using the specified container. Can be used multiple times.
 
 **--label** *label*
 
-Adds an image *label* (e.g. label=*value*) to the image configuration of any
-images which will be built using the specified container.
+Add an image *label* (e.g. label=*value*) to the image configuration of any
+images which will be built using the specified container. Can be used multiple times.
 
 **--os** *operating system*
 
-Specify the target *operating system* for any images which will be built using
+Set the target *operating system* for any images which will be built using
 the specified container.  By default, if the container was based on an image,
 its OS is kept, otherwise the host's OS's name is recorded.
 
 **--port** *port*
 
-Specifies a *port* to expose when running containers based on any images which
-will be built using the specified container.
+Add a *port* to expose when running containers based on any images which
+will be built using the specified container. Can be used multiple times.
```

Comment:
Can you clarify the syntax used in this [Redhat documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/finding_running_and_building_containers_without_docker#creating_images_from_scratch_with_buildah)

````
# buildah config --port 80/tcp working-container
````
Are `/tcp` `/udp` and `/icmp` supported.  If you use this decoration in the config do you have to use that when you launch, for example, a rkt container from the command line?

Reply:
I would doubt you have to do anything special with rkt on the command line, although I have know clue.

@baude @TomSweeneyRedHat Could you verify the syntax.

Reply:
According to the spec the description should be:
> Can be in the format of: `port/tcp`, `port/udp`, `port` with the default protocol being `tcp` if not specified. ICMP is not part of the OCI specification.

Hmm, maybe the documentation burden can be eased by linking to the version of the spec that Buildah is targeting. Example:

> See **OCI spec v1.0.1** [Properties:ExposedPorts](https://github.com/opencontainers/image-spec/blob/v1.0.1/config.md#properties)

If you're happy with that approach I'll add such a reference as I go?

Reply:
Sure lets merge this and then open a PR with these additional changes to docs.